modules/core/src/main/resources/reference.conf

```diff
@@ -44,6 +44,10 @@
   "legacyColumns": []
   "exitOnMissingIgluSchema": true
 
+  "http": {
```

Comment:
This should be:

```
"http": {
  "client": ${snowplow.defaults.http.client}
}
```

...so that we maintain a single set of defaults across all apps.  It references [these defaults](https://github.com/snowplow-incubator/common-streams/blob/a6633b4b533b5ae84f65d94fad93c0841985a59f/modules/runtime-common/src/main/resources/reference.conf#L2-L6) provided by the common-streams `reference.conf`.